source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py

```diff
@@ -144,7 +158,34 @@ def get__from(self, obj):
     additional_informations = EnumListField(attr='additional_informations', pb_type=SectionAdditionalInformationType)
     geojson = SectionGeoJsonField(display_none=False, description='GeoJSON of the shape of the section')
     mode = NestedEnumField(attr='street_network.mode')
-    type = EnumField(display_none=False)
+    type = jsonschema.MethodField(schema_type=lambda: EnumField())
+
+    def get_type(self, obj):
+        def if_on_demand_stop_time(stop):
```

Comment:
hum I would have done a derived EnumField Class for this and add a TODO that the schema type will be false (since it's not strictly the protobu enum as we add values)